process/process.lua

```diff
@@ -359,6 +359,9 @@ function process.handle(msg, _)
     require('.eval')(ao)
   )
 
+  -- AR.IO PROCESS CODE
+  require(".src.init").init()
```

Comment:
this MUST be after _eval - it likely could be placed after boot as well, need to make sure the globals still get created